Wallets

whatston3 opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->

Adds a new slot for wallets to most inventories, and a few new wallet entities to carry cash and a few papers.

Sprites were edited from EricZilla's work on tgstation (https://github.com/tgstation/tgstation/pull/78761), many thanks, it's a nice sprite - we don't need to slot ID cards in, so a center fold should work fine vs. an offset one.

A few items need doing before this is alright:
- [x] Fix abysmal entity definitions
- [x] Parent item slot on jumpsuit
- [x] Add to loadouts
- [x] More sprites?  Pirate wallet?  NT wallet?  Syndie wallet?

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->

Seems fitting, and if you want people to carry cash on them, why not give them a dedicated slot for it?

## How to test
<!-- Describe the way it can be tested -->

TBD - play around with a wallet.

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

Empty wallet slot - there should be (shoddy half-baked) inventory icons for all UI themes.
![image](https://github.com/user-attachments/assets/72b08dab-e419-4843-bf9e-d85aa31f2bfb)

Picture of the wallet open (should really be 3x2 instead of 2x3) - fills show up based on contents of the wallet.
![image](https://github.com/user-attachments/assets/afb9e605-29d4-47dd-b8eb-77c42590f00f)

The same wallet but closed instead of open.
![image](https://github.com/user-attachments/assets/96007ce3-a23a-4121-ac57-b6df21fbd109)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl:
- add: Player characters now have a slot for wallets, check your loadouts for options.

tonotom1 commented:
Begging you to add salvage coins to the storage whitelist, or to add coin purses support to the wallet slot!! I love paying people with the coins I pick up on wrecks.

dvir001 commented:
SR / Mailman should come with a small picture of Clippy

whatston3 commented:
Added mail and pirate captain variants.
Added to loadouts (currently a mandatory free pick).
Separated the base out from the concrete wallets.
Swapped out the purple 500 bill for the blue 10 for contrast.
Added coins to the whitelist, added coin sprites.
Changed the clockwork HUD sprite's shading to be simpler.
Storage changed to 4x2 - should be fine for what's essentially a small clipboard slot that stores cash.

adzee95 commented:
Love it and +1 on the coins!!! I was just thinking, players already have enough pockets and other storages to store these wallets, right? I'm a little unsure if we really need a dedicated slot for it.

whatston3 commented:
> what's the philosophy behind not using the already available storage options?

> I don't see any reason to remove pockets.

Effectively this.  Encourage players to keep cash/documents on hand by giving dedicated storage for it.
It'd be funny if papers put into a wallet looked folded and crumpled.

adzee95 commented:
Yeah, I understand adding the slot to explicitly encourage players to "keep a wallet with money and documents" as part of living on the Frontier, much like the PDA having a dedicated slot. Nice! In my reasoning, the PDA slot would also then not really have a reason to exist, which obviously doesn't make sense.

whatston3 commented:
> should we then maybe also be able to add ID cards to the wallet? seeing as there are some nice card holders in those sprites

No, stick them in your PDA.  I don't want to encourage ID card hoarding in what's essentially a free belt.

adzee95 commented:
Your own ID goes in your PDA, but if you find one, from a roleplay and gameplay perspective, i would argue that it's a little confusing to then not be able to put actual cards into a wallet with (visual) card slots. have we had issues with ID hoarding in the past / do people actually do that? (not familiar with it)

+1 on making papers in the wallet crumpled, it gives more incentive to use a proper document folder or clipboard

dvir001 reviewed (approved):
This is fine as is, hope the "LEGS" wont conflict us later if ever.
